skrub/_sklearn_compat.py

```diff
@@ -0,0 +1,572 @@
+"""Ease developer experience to support multiple versions of scikit-learn.
```

Comment:
can we

- have the URL of where this file came from here
- exclude it from codecov, pre-commit formatting etc. maybe we should have something like a `vendor/` submodule?